code/game/gamemodes/events/powercreeper.dm

```diff
@@ -110,28 +102,40 @@
 /obj/structure/cable/powercreeper/proc/die()
 	grown = 0 //we can't attack or spread anymore
 	do_flick(src, "death[add_state]", 13)
+	updateNeighbours(TRUE)
 	qdel(src)
 
 /obj/structure/cable/powercreeper/proc/try_electrocution(var/mob/living/M)
 	if(!istype(M))
 		return 0
 	playsound(src,'sound/weapons/electriczap.ogg',50, 1) //we still want a sound
-	if(!electrocute_mob(M, powernet, src))
-		M.apply_damage((powernet.avail / 3000), BURN) //one burn damage per 6 plants (keep in mind, this will also take the power from any connected powernet)
+	return electrocute_mob(M, powernet, src)
```

Comment:
removing the burn damage will just mean people can easily cheese the creeper with any kind of insulation. I won't oppose you on a fixed limit on the burn damage to avoid instacrits, but this is stupid

Reply:
I felt it didn't make sense that something that would be presumably immune to electrocution, would still be electrocuted.

Maybe a different negative effect can be thought up to deal with those who are immune to electrocution

Reply:
I would like that, i only added the effect cause it would just mean you can ultracheese it.

Did i mention i don't like insulation code?